.github/workflows/lint-and-tests.yml

```diff
@@ -225,7 +225,7 @@ jobs:
             startsWith(github.event.pull_request.head.ref, 'dependabot/') == false &&
             github.event.pull_request.head.ref != 'chore/crowdin')
         # sha reference has no stable git tag reference or URL. see https://github.com/chromaui/chromatic-cli/issues/797
-        uses: chromaui/action@5f6574e351eb055223ae8ea9e1a734d1d695ea9c
+        uses: chromaui/action@5991575112b472a4fa85429f29cc17e2fb83a2a2
```

Comment:
Why don't we have the version as comment after this, like we do for other actions? It looks like they have tags now? https://github.com/chromaui/action/tags

Reply:
no one checked since I wrote https://github.com/chromaui/chromatic-cli/issues/797#issuecomment-1789941368 - so likely we can add it